### PR TITLE
Adding link to API keys

### PR DESCRIPTION
**Pull request by erikmarr**
## Purpose of this pull request

This pull request (PR) adds a direct link to the API keys topic from the implementation topic.

## Affected DevDocs pages

-  https://devdocs.magento.com/recommendations/implementation.html

### CODE REVIEW

**Review by BarnyShergold — APPROVED** [review]

File: src/recommendations/implementation.md [hunk 1]
```diff
@@ -10,11 +10,11 @@ To help you implement product recommendations on your storefront, use the follow
 
    Deploying Product Recommendations requires two main data sources: [catalog and behavioral]({{ page.baseurl }}/recommendations/product-recs.html#types-of-data). Because production is the only environment where your shoppers' actions can captured and analyzed, it is in your best interest to start data collection on production as early as possible. You can expect higher quality recommendations when you have more data. As an added benefit, when you start collecting behavioral data on production, you can [fetch recommendations based on this production data while operating in non-production environments]({{ page.baseurl }}/recommendations/test.html). This allows you to test and experiment with different recommendations that are computed based on real shopper data collected in production.
 
+   To deploy data collection to production you need to [install and configure the Product Recommendations module]({{ page.baseurl }}/recommendations/install-configure.html) and [generate your API keys](https://docs.magento.com/user-guide/system/saas.html#apikey).
```
Comment (misha-kotov) [inline src/recommendations/implementation.md:13]:
```suggestion
   To deploy data collection to production you need to [install and configure the Product Recommendations module]({{ page.baseurl }}/recommendations/install-configure.html) by [providing an API key](https://docs.magento.com/user-guide/system/saas.html#apikey).
```

Comment (misha-kotov) [reply src/recommendations/implementation.md:13]: trying to avoid the double "and"

**Review by misha-kotov — APPROVED** [review]

File: src/recommendations/implementation.md [hunk 2]
```diff
@@ -10,11 +10,11 @@ To help you implement product recommendations on your storefront, use the follow
 
    Deploying Product Recommendations requires two main data sources: [catalog and behavioral]({{ page.baseurl }}/recommendations/product-recs.html#types-of-data). Because production is the only environment where your shoppers' actions can captured and analyzed, it is in your best interest to start data collection on production as early as possible. You can expect higher quality recommendations when you have more data. As an added benefit, when you start collecting behavioral data on production, you can [fetch recommendations based on this production data while operating in non-production environments]({{ page.baseurl }}/recommendations/test.html). This allows you to test and experiment with different recommendations that are computed based on real shopper data collected in production.
 
+   To deploy data collection to production you need to [install and configure the Product Recommendations module]({{ page.baseurl }}/recommendations/install-configure.html) by [providing an API key](https://docs.magento.com/user-guide/system/saas.html#apikey).
```
Comment (erikmarr) [inline src/recommendations/implementation.md:13]:
```suggestion
   To deploy data collection to production, you need to [install and configure the Product Recommendations module]({{ page.baseurl }}/recommendations/install-configure.html) by [providing an API key](https://docs.magento.com/user-guide/system/saas.html#apikey).
```

**Review by keharper — APPROVED** [review]

**Comment by erikmarr** [comment]
running tests

**Pull request merged**